spec/views/events/index.html.erb_spec.rb

```diff
@@ -2,13 +2,28 @@
 
 RSpec.describe "events/index", type: :view do
   before(:each) do
-    @ws = FactoryGirl.create(:event)
-    assign(:events, [@ws, @ws])
+    @event = FactoryGirl.create(:event)
+    assign(:events, [@event, @event])
   end
 
-  it "renders a list of events" do
+  it "displayes the name" do
     render
-    assert_select "td", :text => @ws.name, :count => 2
-    assert_select "td", :text => @ws.description, :count => 2
+    assert_select "td", :text => @event.name
+  end
+
+  it "displayes the eventkind" do
+    render
+    assert_select "td", :text => @event.kind
+  end
+
+  it "displays the timespan" do
+    render
+    assert_select "td", :date => @event.start_date
+    assert_select "td", :date => @event.end_date
+  end
+
+  it "displayes the status" do
+    render
+    assert_select "td", :boolean => @event.draft
```

Comment:
These tests could all go into a single test. Also, it should read "`displays`".
Good work otherwise 👍